src/EnergyPlus/DataSizing.cc

```diff
@@ -313,6 +313,7 @@ namespace DataSizing {
     int DataDesicDehumNum(0);                       // index to desiccant dehumidifier
     bool DataDesicRegCoil(false);                   // TRUE if heating coil desiccant regeneration coil
     bool HRFlowSizingFlag(false);                   // True, if it is a heat recovery heat exchanger flow sizing
+    bool SizeHRHXtoMinFlow(false);                  // true, if the heat recovery heat exchanger nominal flow is sized to min OA flow
```

Comment:
New flag variable is used to decide the of use minimum or maximum OA flow for sizing Heat Recovery HX.